packages/service/src/plugin.ts

```diff
@@ -82,15 +150,17 @@ export class PluginsRunner {
     fileName: string,
     allChangedFiles: Set<string>
   ): AsyncGenerator<Set<string>> {
-    for (const plugin of this.plugins) {
-      if (plugin.options.filter) {
-        if (!plugin.options.filter(fileName)) {
+    for (const level of this.levels) {
```

Comment:
What is levels in this context of plugins. We could have `n` layers of plugins to apply?

Is this for multiple mixed contexts, glint first level, then JS next level?